README.md

```diff
@@ -82,26 +69,25 @@ To get the name of the device enter:
     emulator -list-avds
 
 Then open one terminal and run the command:
-    react-native start
-
-This terminal will prevent an error on run.
+    
+    `npm start`
 
 Then open another terminal and run:
-    react-native run-android
+
+    `npm run android`
 
 This may take a while especially the first time.
 If the app doesn't open automatically go to apps on the device and click AlumniApp.
-You should see a welcome message.
 
 ## Common errors
 
 ### No tools.jar
-Linux distros often use open java sdk which react does not like and it will complain
-about not having a "tools.jar". I have heard of people getting around this by making sure
-there is the dev or devel tools involved for opensdk but I suggest installing [oracle's jdk.](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-Then make sure the JAVA_HOME variable points to oracle's jdk using:
-    export JAVA_HOME=path
-
-### UNMET PEER DEPENDANCY
```

Comment:
Unmet peer dependencies only occur if someone committed erroneous codes with invalid `package.json` files. Assuming contributors follow the convention of adding dependencies with `npm install --save <package...>` or `npm install --save-dev <package...>` this should not be an issue.